python/lsst/daf/butler/datastores/s3Datastore.py

```diff
@@ -38,6 +42,9 @@
     Union,
 )
 
+# https://pypi.org/project/backoff/
+import backoff
```

Comment:
Do we want to try to protect against backoff not being present or just assume that it is?

Reply:
Added a no-op decorator for that protection.